projects/module-3/rancid-tomatillos-v3.md

```diff
@@ -79,11 +79,13 @@ In the future, when we use the actual API to get individual movie details, the i
 
 ### Iteration 3 - Network Requests & Async JS
 
-For the third iteration, we'll refactor our application to use actual data from the database (information is below) instead of our mocked `movieData` file. We'll also add some type checking with [PropTypes](https://frontend.turing.edu/lessons/module-3/proptypes.html)
```

Comment:
GitHub is making it look like I deleted the PropTypes note. I didn't it's still in the file. I don't know why this is red.